utility-emissions-channel/typescript_app/runManual.sh

```diff
@@ -6,8 +6,14 @@ rm -rf ./node_modules
 echo "Updating npm..."
 npm update
 
+echo "Setting user to root..."
+npm config set user root
+
+echo "Check current user..."
+id -un
+
 echo "Installing node modules..."
-npm install
+npm install -g
```

Comment:
@Pihu1998 do we need to install these packages globally ? can we just not put these pkgs in `devDependencies` and use command like `npx tsc` , `npx serverless`. or better if we put realted commands inside npm scripts. like https://github.com/Zzocker/blockchain-carbon-accounting/blob/4a24b70ed746df4362a7e1389408fc0fd9a37614/utility-emissions-channel/typescript_app/package.json#L11